### PR TITLE
feat(cli/build): add windows manifest

### LINKED ISSUE #9862: `Deno.osRelease()` in Windows
hi guys I think there is a problem extracting the OS version via the `Deno.osRelease()` method in Windows because when I run this code,
```ts
console.log(Deno.osRelease());
```
 I get `6.2.9200` in the console even though I am using Windows 10

Deno: v1.8.1
Windows: 10.0.19042

### PR DESCRIPTION
**Pull request by littledivy**
Fixes #9862

### CODE REVIEW

File: cli/manifest.xml
Comment (lucacasonato) [inline cli/manifest.xml:6]: This is going to be a problem. Can you set it up so this is replaced in build.rs automatically by the correct version?

**Comment by littledivy** [comment]
Not sure what the issue is here :thinking: : https://github.com/denoland/deno/pull/9870/checks?check_run_id=2183404333#step:27:16

**Comment by bartlomieju** [comment]
What problem does this PR fix?

I will defer to Bert's review before landing

**Comment by littledivy** [comment]
> What problem does this PR fix?

Adds a manifest for Deno which fixes the `DenoosRelease()` behavior on Windows. See https://github.com/denoland/deno/issues/9862#issuecomment-804827083

**Review by piscisaureus — APPROVED** [review]

**Comment by ry** [comment]
Closing because stale. Open a new PR if you want to still get this change in.

**Pull request closed without merging**
